### DISCUSSION TITLE
Merging all (most) of the branches?

### DISCUSSION THREAD

**Discussion by eva0marques** [Spatiotemporal-Exposures-and-Toxicology/NRT-AP-Model#173] [General]
@Spatiotemporal-Exposures-and-Toxicology @dzilber @sigmafelix @dawranadeep @MAKassien @Sanisha003 @larapclark 
Maybe we should discuss this afternoon about regularly merging all of the branches. I feel it is starting to become a little bit messy and it's getting hard to find our way around. I know we are waiting for the end of milestones to do so, but I think that the more we are waiting the harder it will be to merge everything, no?

**Comment by kyle-messier** [comment]
@eva0marques This is a good idea. I was planning to discuss our plans for wrapping up the TDD testing phase of the project. We can discuss this today too as we try to check off milestones.

**Comment by kyle-messier** [comment]
We're down to 7 branches!

**Reply by eva0marques** [reply]
Good job! Thank you!!

**Comment by sigmafelix** [comment]
@Spatiotemporal-Exposures-and-Toxicology @dzilber @eva0marques @dawranadeep @MAKassien @Sanisha003 @larapclark
I want to recommend you merge the current `main` into your working branch or start a new branch from the current `main` if possible. I fixed dependency and lint errors there.
